fvm/evm/evm_test.go

```diff
@@ -140,8 +143,8 @@ func TestEVMAddressDeposit(t *testing.T) {
                                    address.deposit(from: <-vault)
                                }
                             `,
-							chain.ServiceAddress().HexWithPrefix(),
-							fvm.FlowTokenAddress(chain).HexWithPrefix(),
+							sc.FlowServiceAccount.Address.HexWithPrefix(),
```

Comment:
@ramtinms @turbolent I found this to be curious. The import imports the EVM contract, which should be on a different address, not on the System address, at least according to `EVMAccountIndex`.

Reply:
There are two separate addresses (accounts) here: 
- The address where the EVM contract is deployed to. This can be any address. Given that it gets special permissions to call into FVM's internal EVM functionality, we deemed it best to deploy it to the service account
- The address where the EVM state is stored (`EVMAccountIndex`)
The two can, but do not have to be the same, and are currently the same.